.github/workflows/ci.yml

```
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.11", "3.12"]
+        python-version: ["3.11", "3.12", "3.13"]
```

```suggestion
        # remember to update the envlist in tox.ini too
        python-version: ["3.11", "3.12", "3.13"]
```
Maybe this comment is enough.